uiowa/uiowa
# Minor UI Press update to remove the need for overridden page EVD

All blocks in the Site Custom list are allowed, so putting blocks into that category doesn't require overriding the page EVD config. That was the only difference in the page EVD config for UI Press. So this update makes complete config split no longer necessary.

# How to test

```
ddev blt ds --site uipress.uiowa.edu && ddev drush @uipress.local uli node/1/layout
```
- Confirm no issues with sync.
- Click 'Add component' somewhere on the page. Click 'More...'
- Confirm that 'Featured Books' shows up in the 'Site Custom' section.
- Confirm that block still shows up under 'New and Noteworthy' on https://uipress.uiowa.ddev.site/
